crates/astria-sequencer/src/sequence/mod.rs

```diff
@@ -1,6 +1,7 @@
 pub(crate) mod action;
 pub(crate) mod component;
 mod state_ext;
+pub(crate) mod storage;
```

Comment:
I updated the visibility of the types inside this module in [26e3683](https://github.com/astriaorg/astria/pull/1492/commits/26e36839ad833bd65b420e6f080b14b20cb4acee).  I kept the `storage` module `pub(crate)` so that consumers of the `Value` need to do `use crate::sequence::storage::Value` rather than `crate::sequence::Value`, since that's a bit vague.

`Value` is the only type exported at `pub(crate)` visibility from the `storage` module for all components.  Happy to change this if you want (e.g. I could make the `storage` module private, and export the `Value` using an alias).

Reply:
No, this makes sense and I would have probably done the same, thank you for context!